scripts/upload_pact.sh

```diff
@@ -5,3 +5,8 @@ curl --fail --show-error -v -XPUT \-H "Content-Type: application/json" \
   -d@$PROJECT_LOCATION/$LIBRARY_MODULE/target/pacts/access-checkout-android-sdk-verified-tokens.json \
   -u $PACTBROKER_USERNAME:$PACTBROKER_PASSWORD \
   https://$PACTBROKER_URL/pacts/provider/verified-tokens/consumer/access-checkout-android-sdk/version/$PROJECT_VERSION
+
+curl --fail --show-error -v -XPUT \-H "Content-Type: application/json" \
+  -d@$PROJECT_LOCATION/$LIBRARY_MODULE/target/pacts/access-checkout-android-sdk-sessions.json \
+  -u $PACTBROKER_USERNAME:$PACTBROKER_PASSWORD \
+  https://$PACTBROKER_URL/pacts/provider/sessions/consumer/access-checkout-android-sdk/version/1.0.0
```

Comment:
should this make use of a variable at the top of the file such as
export SESSIONS_CONTRACT_VERSION="1.0.0"

and then referenced as

/version/$SESSIONS_CONTRACT_VERSION

Reply:
in addition we could do the same for VT one and change it to
export $VT_CONTACT_VERSION = $PROJECT_VERSION

for readability?

Reply:
Thanks, I've made those changes